infrastructure/palo-alto.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  trusted_vnet_name           = "core-infra-vnet-${var.env}"
+  trusted_vnet_resource_group = "core-infra-${var.env}"
+  trusted_vnet_subnet_name    = "palo-trusted"
+}
+
+module "palo_alto" {
```

Comment:
I would be quite wary of having the palo alto infrastructure in an application repository,
the automation is quite poor, runs on every build and sometimes fails

you probably want the infrastructure provisioned in a separate  repo just for infrastructure?

Reply:
ah. so here comes the culprit which requires separate infra. will it be safe to move out all the rest as well? good thing nothing is used atm

Reply:
Agree with @timja else we would getting blocked with every build in future

Reply:
ok. will extract what's current and raise this in new repo

Reply:
it's this guy + the DNS challenges required  for a custom domain on the storage account can be quite a pain...